### DISCUSSION TITLE
Flows:  flows will auto set `user_updated` to `null`

### DISCUSSION THREAD

**Discussion by ddiu8081** [directus/directus#15345] [Archive]
Hi,
I created some flow to update items (cron jobs that has no trigger-user, and event-hook jobs that has trigger-user). 

But I found they will always set the `user_updated` field to `null`. And I can't find the change log in Revisions panel either:

<img width="934" alt="ScreenShot 2022-09-01 at 02 07 13@2x" src="https://user-images.githubusercontent.com/1998168/187748904-e18be3be-6d5f-4b95-8ad5-5f17594efae6.png">

![ScreenShot 2022-09-01 at 02 01 56@2x](https://user-images.githubusercontent.com/1998168/187747919-180f25d9-6ef2-4d27-9f06-1d0052044b43.png)

(In revisions panel, only records modified directly by the user can be found, not those modified by the system.)

Can the `user_updated` field be set to other value than `null`, like a system user? And can the changelog can be show in revisions?

Thanks!

**Comment by rijkvanzanten** [comment]
> Can the user_updated field be set to other value than null, like a system user? 

There are no "system users" by default, and there's currently no way to configure what "user" the flow pretends to be during updates, so this is a new feature request

> And can the changelog can be show in revisions?

Good one! I can't think of a reason why that doesn't work currently. Mind opening a new Issue? I think that's a bug 🙂

**Comment by teichsta** [comment]
same issue here … so what is the preferred way to work with the system fields (updated date and user updated) when using flows?

**Reply by ddiu8081** [reply]
revisions tracking bugs are fixed in issue #15346, and met my needs
